docs/gallery/index.ipynb

```diff
@@ -33,7 +33,8 @@
     "* [3-D scatter plot with threshold](scatter3d-with-threshold.ipynb)\n",
     "* [3-D scatter plot with slider](scatter3d-with-slider.ipynb)\n",
     "* [Interactive masking](interactive-masking.ipynb)\n",
-    "* [Peeling off the layers](peeling-layers.ipynb)"
+    "* [Peeling off the layers](peeling-layers.ipynb)\n",
+    "* [Titled random sample histograms](tiled-random-samples.ipynb)"
    ]
```

Comment:
I find it is a shame that it's the last figure which appears as the preview in the gallery page:
![Screenshot at 2023-12-14 09-39-03](https://github.com/scipp/plopp/assets/39047984/8f218007-75da-48fe-a568-87589b694c3a)

It would look more 'impressive' if it was the first figure. Can you save the first figure as a thumbnail (in a hidden cell at the end of the notebook maybe?) and add it to the `docs/conf.py` (see other notebooks in the gallery for an example of how this is done).